network-api/networkapi/wagtailpages/factory/bannered_campaign_page.py

```diff
@@ -35,11 +35,11 @@ def generate(seed):
     reseed(seed)
 
     try:
-        bannered_campaign_namespace = WagtailPage.objects.get(title='campaigns')
-        print('Campaigns namespace exists')
+        bannered_campaign_index_page = WagtailPage.objects.get(title='campaigns')
+        print('campaign index page exists')
     except WagtailPage.DoesNotExist:
-        print('Generating campaigns namespace')
-        bannered_campaign_namespace = MiniSiteNamespaceFactory.create(
+        print('Generating a campaign index page')
+        bannered_campaign_index_page = MiniSiteNamespaceFactory.create(
```

Comment:
If we're updating this file, let's stop building a minisite here, since we're using a CampaignIndexPage on prod now (so the same as `campaign_page.py`)

Reply:
ah def didn't catch these as I was just doing find & replace for  `campaign_namespace` to `campaign_index_page` lol 😂